### PR TITLE
Remove SPS/Token migration code. Remove GHES url manipulate code.

### PR DESCRIPTION
**Pull request by TingluoHuang**
SPS has been power down, so I can delete all the migration code.

GHES cleanup:
https://github.com/github/c2c-actions-runtime/issues/483

### CODE REVIEW

File: src/Runner.Listener/MessageListener.cs [hunk 1]
```diff
@@ -13,10 +13,7 @@
 using System.Runtime.InteropServices;
```
Comment (TingluoHuang) [inline src/Runner.Listener/MessageListener.cs:13]: All changes that are related to SPS/Token conversion is removed from this file.

File: src/Runner.Common/ConfigurationStore.cs [hunk 2]
```diff
@@ -232,21 +232,6 @@ public void SaveCredential(CredentialData credential)
             File.SetAttributes(_credFilePath, File.GetAttributes(_credFilePath) | FileAttributes.Hidden);
         }
 
-        public void SaveMigratedCredential(CredentialData credential)
```
Comment (TingluoHuang) [inline src/Runner.Common/ConfigurationStore.cs:235]: `SaveMigratedCredential` was used when we migrate from SPS to Token.

File: src/Runner.Common/RunnerServer.cs [hunk 3]
```diff
@@ -50,10 +50,6 @@ public interface IRunnerServer : IRunnerService
 
         // agent update
         Task<TaskAgent> UpdateAgentUpdateStateAsync(int agentPoolId, int agentId, string currentState);
-
-        // runner authorization url
```
Comment (TingluoHuang) [inline src/Runner.Common/RunnerServer.cs:54]: no longer need these.

File: src/Runner.Listener/Configuration/ConfigurationManager.cs [hunk 4]
```diff
@@ -221,44 +232,18 @@ public async Task ConfigureAsync(CommandSettings command)
...
-                UriBuilder configServerUrl = new UriBuilder(runnerSettings.ServerUrl);
-                UriBuilder oauthEndpointUrlBuilder = new UriBuilder(agent.Authorization.AuthorizationUrl);
                 var credentialData = new CredentialData
                 {
                     Scheme = Constants.Configuration.OAuth,
                     Data =
                     {
                         { "clientId", agent.Authorization.ClientId.ToString("D") },
                         { "authorizationUrl", agent.Authorization.AuthorizationUrl.AbsoluteUri },
-                        { "oauthEndpointUrl", oauthEndpointUrlBuilder.Uri.AbsoluteUri },
```
Comment (TingluoHuang) [inline src/Runner.Listener/Configuration/ConfigurationManager.cs:261]: VssOAuthCredential.cs will use `authorizationUrl` as `oauthEndpointUrl` if `oauthEndpointUrl` is not set.

File: src/Runner.Listener/Program.cs [hunk 5]
```diff
@@ -102,7 +102,9 @@ private async static Task<int> MainAsync(IHostContext context, string[] args)
                 IRunner runner = context.GetService<IRunner>();
                 try
                 {
-                    return await runner.ExecuteCommand(command);
+                    var returnCode = await runner.ExecuteCommand(command);
+                    trace.Info($"Runner execution has finished with return code {returnCode}");
```
Comment (TingluoHuang) [inline src/Runner.Listener/Program.cs:106]: add a trace to show the return code of the runner.

File: src/Runner.Listener/Configuration/ConfigurationManager.cs [hunk 6]
```diff
@@ -119,6 +119,17 @@ public async Task ConfigureAsync(CommandSettings command)
                     // Determine the service deployment type based on connection data. (Hosted/OnPremises)
                     runnerSettings.IsHostedServer = runnerSettings.GitHubUrl == null || IsHostedServer(new UriBuilder(runnerSettings.GitHubUrl));
 
+                    // Warn if the Actions server url and GHES server url has different Host
```
Comment (ericsciple) [inline src/Runner.Listener/Configuration/ConfigurationManager.cs:122]: nice

Comment (ericsciple) [reply src/Runner.Listener/Configuration/ConfigurationManager.cs:122]: Chatted offline...

Example `actionsServerUrl` is `https://my-ghes/_services/pipelines/[...]`
Example `githubServerUrl` is `https://my-ghes`

Consider adding ^ example into the comments in the source code

Should we just fail? Should use hosts file, or DNS, or proxy bypass to fix routing. Server doesn't support multiple host names.

File: src/Runner.Listener/Configuration/ConfigurationManager.cs [hunk 7]
```diff
@@ -221,44 +232,18 @@ public async Task ConfigureAsync(CommandSettings command)
             // Add Agent Id to settings
             runnerSettings.AgentId = agent.Id;
 
-            // respect the serverUrl resolve by server.
```
Comment (ericsciple) [inline src/Runner.Listener/Configuration/ConfigurationManager.cs:224]: 🎉

File: src/Runner.Listener/Configuration/ConfigurationManager.cs (hunk 4 above)
Comment (ericsciple) [reply src/Runner.Listener/Configuration/ConfigurationManager.cs:261]: chatted offline... previously authorizationUrl was audience (claim) and oauthEndpointUrl was where to connect (SPS)

File: src/Runner.Listener/MessageListener.cs (hunk 1 above)
Comment (ericsciple) [reply src/Runner.Listener/MessageListener.cs:13]: discussed offline... diff new code against pre-migration code === identical

**Pull request merged**